exonum/src/storage/rocksdb.rs

```diff
@@ -59,7 +64,10 @@ struct RocksDBIterator {
 }
 
 impl RocksDB {
-    /// Open a database stored in the specified path with the specified options.
+    /// Opens a database stored at the specified path with the specified options.
+    /// If the database does not exist at the indicated path and the option
+    /// 'create_if_missing' is switched on in 'DBOptions', a new database will
```

Comment:
Please use code formatting for `create_if_missing` and `DbOptions` (the latter is misspelled, btw).